core/src/main/scala/org/apache/spark/storage/ShuffleBlockFetcherIterator.scala

```diff
@@ -1037,39 +1037,48 @@ final class ShuffleBlockFetcherIterator(
       blockId: BlockId): String = {
     logInfo("Start corruption diagnosis.")
     val startTimeNs = System.nanoTime()
```

Comment:
This line could move to `ShuffleBlockId` case only.

Reply:
Updated